samples/system-test/recognize.v1p1beta1.test.js

```diff
@@ -25,13 +25,16 @@ const {runAsync} = require(`@google-cloud/nodejs-repo-tools`);
 const bucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
 const cmd = `node recognize.v1p1beta1.js`;
 const cwd = path.join(__dirname, `..`);
-const filename = `Google_Gnome.wav`;
-const filepath = path.join(__dirname, `../resources/${filename}`);
-const text = `the weather outside is sunny`;
+const filename1 = `Google_Gnome.wav`;
+const filename2 = `commercial_mono.wav`;
```

Comment:
I can't find this being used anywhere? Why was this variable added to the tests?

If `filename2` and `filepath2` and `test2` aren't used, the changes here should be reverted back to the original `filename`, `filepath`, and `text` vars

Reply:
Nevermind, found it, missed it the first time